source/includes/_aviate-wallet.md

```diff
@@ -0,0 +1,370 @@
+
+
+---
+
+### Wallet
+
+Represents a wallet associated with a customer account. It maintains the available and pending credit balances, top-off configuration, and credit history records.
+
+| Name           | Type                 | Generated by | Description                                                              |
+|----------------|----------------------|--------------|--------------------------------------------------------------------------|
+| `walletId`     | `string` (UUID)      | system       | Unique identifier for the wallet                                         |
+| `kbAccountId`  | `string` (UUID)      | user         | Kill Bill Account ID associated with the wallet                          |
+| `currency`     | `string`             | user         | Currency of the wallet (must match the account's default currency)       |
+| `balance`      | `string`             | system       | Current available credit balance                                         |
+| `liveBalance`  | `string`             | system       | Balance including credits reserved for the next invoice                  |
+| `topOff`       | [`TopOff`](#topoff)  | user         | Configuration for automatic wallet replenishment                         |
+| `records`      | `WalletRecord[]`     | system       | Ledger entries for credit additions and consumptions                     |
+
+---
+
+### WalletRecord
+
+Represents a credit transaction in the wallet ledger, including original and remaining amounts, optional expiration, and linkage to invoice/payment.
+
+| Name             | Type           | Generated by | Description                                                              |
+|------------------|----------------|--------------|--------------------------------------------------------------------------|
+| `recordId`       | `integer`      | system       | Unique identifier for the ledger entry                                   |
+| `creditType`     | `string`       | user         | Type of credit (`CREDIT_PAID`, `CREDIT_FREE`, `CREDIT_USED`)             |
+| `originAmount`   | `string`       | system       | Original credited amount                                                 |
+| `remainAmount`   | `string`       | system       | Remaining unused portion of the credit                                   |
+| `description`    | `string`       | user         | Optional description or memo                                             |
+| `expDate`        | `string` (ISO datetime) | user  | Optional expiration timestamp                                          |
+| `parentRecordId` | `integer`      | system       | Reference to the original credit record (for partial consumption)        |
+| `kbInvoiceId`    | `string` (UUID)| system       | Associated Kill Bill invoice ID (if credit was paid)                     |
+| `kbPaymentId`    | `string` (UUID)| system       | Associated Kill Bill payment ID                                          |
```

Comment:
`(if credit was paid)`?

Reply:
Fixed in [fc91c9a](https://github.com/killbill/slate/pull/339/commits/fc91c9a285f367ff28d62efbbfb0554d11a261ee)